resources/schema.json

```diff
@@ -112,6 +112,13 @@
             "additionalProperties": false,
             "description": "Contains the settings for different mutations and profiles",
             "properties": {
+                "ignore": {
```

Comment:
Something like `ignore-all` can't be a mutator name, doesn't have to be reserved or tested against.

Reply:
`global-ignore`?

Reply:
Yes, that'd be fine too. Everything with a dash works like that.

Reply:
went for `global-ignore` since it's more in line with the names in `MutatorResolver`